### ISSUE TITLE
Is this for Arduino YUN Rev-2??

### ISSUE THREAD

**Issue by nsmith1024** [arduino/lede-yun#6]
Hello,

Im trying to find the official OpenWRT for Arduino YUN rev-2 hardware?

Is this the one, or can you tell me the git hub link to it?

Thanks

**Comment by facchinm** [comment]
It's this one; if you just want the feed packages for recompiling vanilla openwrt you can add https://github.com/arduino/openwrt-arduino-packages to your `feeds.conf`

**Comment by nsmith1024** [comment]
Do you know where is the repository for the OpenWRT Arduino YUN rev-2 board? Not the packages, i mean the linux OS code itself for the rev-2 board?

**Comment by facchinm** [comment]
This exact repository is the one you are looking for. Please stop spamming all the other repos

**Comment by nsmith1024** [comment]
Sorry i was not trying to spam, i ask so many forums and nobody answered, i even asked on the arduino forum itself, nobody answered, so i started asking around.

**Comment by nsmith1024** [comment]
> 
> 
> This exact repository is the one you are looking for. Please stop spamming all the other repos

I tried building openWRT for YUN using this repo but it has a lot of missing dependencies

1. What is the difference between this repo and http://github.com/openwrt/openwrt?

I tried using http://github.com/openwrt/openwrt, it also has Arduino YUN already in the settings if you select target=ARTH79 so why do we need two repos for OpenWRT YUN?

2. However when i build that version of OpenWRT the build size is only 5.4M, isnt it supposed to be about 16M?

Any ideas why their build is only 5.4M?

openwrt-generic-arduino_yun-squashfs-sysupgrade.bin is only 5.4M
